remoteworkitem/trackerquery_repository_test.go

```diff
@@ -104,7 +104,7 @@ func (test *TestTrackerQueryRepository) TestExistsTrackerQuery() {
 func (test *TestTrackerQueryRepository) TestTrackerQuerySave() {
 	t := test.T()
 	resource.Require(t, resource.Database)
-
+	test.DB.LogMode(true)
```

Comment:
Forgot to remove it after debugging?
I'm not against additional logs in tests though ;)

Reply:
ah right, let me remove that statement, I just needed it when I was trying to understand why the test failed ;)